py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py

```diff
@@ -329,6 +329,41 @@ def _populate_trt_builder_config(
         if self.compilation_settings.enable_weight_streaming:
             builder_config.set_flag(trt.BuilderFlag.WEIGHT_STREAMING)
 
+        if version.parse(trt.__version__) >= version.parse("10.8"):
```

Comment:
We can just drop 10.7 instead having this piecemeal support

Reply:
Looks like we do it for some settings but not others, so we need to decide if we want versioned builder config or not

Reply:
my default stance is no but if its not too much work (outside of 2.7 scope) then we might